src/crystal/event_loop.cr

```diff
@@ -1,25 +1,19 @@
-require "event"
+require "./event"
 
-# :nodoc:
-class Scheduler
-  @@runnables = Deque(Fiber).new
-  @@eb = Event::Base.new
+class Crystal::EventLoop
```

Comment:
Why not a `module`?

Reply:
Probably because Scheduler used to be a class with only class methods too, and I overlooked that.

Reply:
would it be possible to keep it as a class, and make all current class methods → instance methods, and ultimately allow to make multiple event loops? (no idea what usecase there could possibly be for that though)

Reply:
I don't have any plans to experiment with many event loops. Quite the opposite.